ckan/logic/action/create.py

```diff
@@ -1032,10 +1035,18 @@ def user_create(context: Context,
 
     # Create dashboard for user.
     dashboard = model.Dashboard(user.id)
+
     session.add(dashboard)
     if not context.get('defer_commit'):
         model.repo.commit()
 
+    if with_apitoken:
+        # Create apitoken for user.
+        api_token = _get_action("api_token_create")(
+            context, {"user": user.name, "name": "default"}
```

Comment:
It will cause an AuthorizationError because you are potentially using anonymous context

Reply:
@smotornyuk done